docker-compose.yml

```diff
@@ -1,12 +1,18 @@
 services:
-  backend:
-    build: 
-      context: infrastructure/docker/backend
+  cd-service:
+    build:
+      context: .
+      dockerfile: services/cd-service/Dockerfile
       args:
         - UID=$USER_UID
-    container_name: kuberpult-cd-service
+        - DIR=services/cd-service
+        - PARENT_CONTAINER=latest
```

Comment:
The dockerfiles expect `PARENT_CONTAINER` to be the full name of the image not `latest`

Reply:
done